conda/recipes/numbast/meta.yaml

```diff
@@ -32,6 +51,7 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
```

Comment:
Should we do:
```suggestion
    - setuptools # [py_version == '313']
```
or that's unnecessary?

Reply:
I would make it unconditional, it was already a dependency and it is now explicit and I assume 3.14 and later would also need it.